### ISSUE TITLE
Highlight issue, when migrated to latest 2.0.0 it become

### ISSUE THREAD

**Issue by ajay9889** [readium/r2-testapp-kotlin#424]
In 2.0.0-beta.1 it was not the issue. But, when migrated it reappears the issue upon migrating to 2.0.0. 

Before adding the highlights in the database, the highlight was working fine: Please refer to below screen. It is correctly placed when adding into the Highlight database.

![Screenshot_1621741567](https://user-images.githubusercontent.com/9216373/119247500-93a14e80-bbbc-11eb-97c4-d949524c14c6.png)

But, when clicking and navigating from the highlight list it is getting in the wrong place. 

![Screenshot_1621741586](https://user-images.githubusercontent.com/9216373/119247541-f266c800-bbbc-11eb-8778-49e7df9e6cdc.png)

**Comment by mickael-menu** [comment]
I can't reproduce this issue in `r2-testapp-kotlin`, did you?

**Comment by ajay9889** [comment]
No, the highlight issue is still there in the test app also when are using our ePub book in which has the chapter link. Kindly see the recorded video.  Please check the actual root cause so that we can fix it as we have lined up a pilot launch.

https://user-images.githubusercontent.com/9216373/120606548-e236ce80-c481-11eb-91cf-5296ee375212.mp4

https://user-images.githubusercontent.com/9216373/120606592-ee229080-c481-11eb-8cda-beb62d439477.mp4

Epub Book:

[nck_epub.zip](https://github.com/readium/r2-testapp-kotlin/files/6589808/nck_epub.zip)

**Comment by mickael-menu** [comment]
@ajay9889 I'm sorry but I don't have any time planned for the current highlights implementation, you will need to find a workaround or a bugfix yourself.

**Comment by parsakhosravani** [comment]
Just add this code in line 66 of the highlight.js file

`document.addEventListener("DOMContentLoaded", function(event) {
      setScrollMode(false);
 });`
